draft/spec/index.html

```diff
@@ -953,6 +965,127 @@ <h2>Versioned OCFL Object</h2>
 
     <section id="example-bagit-in-ocfl">
         <h2>BagIt in an OCFL Object</h2>
+        <p>
+            [[BagIt]] is a popular file packaging specification, but unlike OCFL it does not provide a mechanism for
+            content versioning. Using OCFL it is possible to store a BagIt structure with content versioning, such
+            that when the object state is resolved, it creates a valid BagIt 'bag'. This example will illustrate how
+            this can be accomplished, using the <a href="https://tools.ietf.org/html/draft-kunze-bagit-17#section-4.1">
+            example of a basic bag</a> given in the BagIt specification.
+        </p>
+<pre>
+[object root]
+    ├── 0=ocfl_object_1.0
+    ├── inventory.json
+    ├── inventory.json.sha512
+    └── v1
+        └── myfirstbag
+            ├── bagit.txt
+            ├── data
+            │   └── 27613-h
+            │       └── images
+            │           ├── q172.png
+            │           └── q172.txt
+            └── manifest-md5.txt
+</pre>
+
+        <p>
+            If, for example, a new directory were added in a subsequent version, the OCFL Object would look like this:
+        </p>
+<pre>
+[object root]
+    ├── 0=ocfl_object_1.0
+    ├── inventory.json
+    ├── inventory.json.sha512
+    ├── v1
+    │   └── myfirstbag
+    │       ├── bagit.txt
+    │       ├── data
+    │       │   └── 27613-h
+    │       │       └── images
+    │       │           ├── q172.png
+    │       │           └── q172.txt
+    │       └── manifest-md5.txt
+    └── v2
+        └── myfirstbag
+            ├── data
+            │   └── 27614-h
+            │       └── images
+            │           ├── q173.png
+            │           └── q173.txt
+            └── manifest-md5.txt
+</pre>
+        <p>
+            The state of the object at version 2 would resemble the following BagIt object:
```

Comment:
"resemble" sounds to weak to me. the state really **is** that Bagit object. Perhaps "be"  or "match"?